packages/combobox/src/useCombobox.ts

```diff
@@ -716,13 +713,12 @@ export const useCombobox = <
   );
 
   const getListboxProps = useCallback<IUseComboboxReturnValue['getListboxProps']>(
-    ({ role = 'listbox', 'aria-labelledby': ariaLabeledBy = null, ...other }) =>
-      getDownshiftListboxProps({
+    ({ role = 'listbox', ...other }) =>
+      getDownshiftListboxProps<any>({
```

Comment:
Just checking re: `any` so I'm on the same page - is the intent here is to defer all typing to Garden's `getListboxProps` and prevent transient TS errors that may confuse consumers?

Reply:
Sort of. imo, while well-intended, the associated [Downshift PR](https://github.com/downshift-js/downshift/pull/1482) kind of made a mess out of the getter prop types to the point where there was no reasonable combo that could satisfy what is going on – so I hit the `any` eject button. At least we're coercing the options object below to be of a valid Downshift type for the function call. But this is all deeply internal to the hook code and has no implications for the well-typed external API.